kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/README.md

```diff
@@ -6,29 +6,33 @@ This instruction describes all steps to install KIE Server on Tomcat 9 standalon
  2. Copy following libraries into TOMCAT_HOME/lib
    - javax.security.jacc:javax.security.jacc-api
    - org.kie:kie-tomcat-integration
-   - org.slf4j:artifactId=slf4j-api
-   - org.slf4j:artifactId=slf4j-jdk14
+   - org.slf4j:slf4j-api
+   - org.slf4j:slf4j-jdk14
    - org.jboss.spec.javax.transaction:jboss-transaction-api_1.2_spec
    - org.jboss.integration:narayana-tomcat
    - org.jboss.narayana.jta:narayana-jta
    - org.jboss:jboss-transaction-spi
+   - org.jboss.logging:jboss-logging
 
  versions of these libraries will depend on the release, so best to check what versions are shipped with KIE
```

Comment:
Here are all the versions defined https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L56

Reply:
@mswiderski do you want me to make the update in my fork?

Reply:
yes, please go ahead